docs/help-center/your-account/account-settings.mdx

```diff
@@ -489,6 +489,33 @@ Depending on the state of the chat backup, you have two options:
 
 In both cases, make sure to store the generated passphrase in a secure place.
 
+## Chat settings
+
+### Change the behavior of the Enter key
+
+You can alter the press Enter action in this section. This setting is per device. After logout it will be set to default action again.
```

Comment:
This doesn't sound very nice.
Probably better:
```
In this section you can find information on how to alter the behaviour of the Enter key.

:::info
This setting is per device and will be set to default on logout. You will have to set it again every time you log in.
:::
```

Reply:
Thank you so much. Fixed